deploy/docs/opentelemetry_collector.md

```diff
@@ -47,11 +49,26 @@ All Opentelemetry Collector configuration for metrics is located in
 If you want to modify it, please see [Sumologic Opentelemetry Collector configuration][configuration]
 for more information.
 
-## Logs Metadata
+## Logs
 
-We are using Opentelemetry Collector like Fluentd to enrich metadata and to filter data.
+Opentelemetry Collector can be used for both log collection and metadata enrichment. For these roles,
+it replaces respectively Fluent-Bit and FluentD.
```

Comment:
Micro-nit: I think it's "Fluentd", not "FluentD" :)

Reply:
Also I believe Fluent Bit does not use a hyphen in the name: https://fluentbit.io/
<img width="289" alt="image" src="https://user-images.githubusercontent.com/70892616/186662242-fcaf3b9d-00b5-410b-97c5-e7cb8bf2141b.png">

Reply:
Fixed